src/components/PopupWithFormConfirmDelete.js

```diff
@@ -0,0 +1,24 @@
+import PopupWithForm from "./PopupWithForm.js";
+
+export default class PopupWithFormConfirmDelete extends PopupWithForm {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.